app/assets/javascripts/Patient.js

```diff
@@ -0,0 +1,143 @@
+var Schema = mongoose.Schema;
+var Code = Schema.Types.Code;
+var Interval = Schema.Types.Interval;
+var Quantity = Schema.Types.Quantity;
+
+var PatientSchema = new Schema({
+  birth_datetime: DateTime,
```

Comment:
Is this missing the name fields?

Reply:
Good catch!

Reply:
what about expected values? and cached results?

Reply:
should we create a data structure (where?) for result types? hash keyed by population, and then keyed by IPP/DENOM/MSRPOPLEX/etc.?

Reply:
Yeah, this is something I didn't think through enough. We definitely should _at least_ have a Result object type.

So maybe have an `Expected`, which a patient has one of for each population (and by default has one that has nothing expected to be passing), and also a `Results` object that a patient can have for each population. Ideally they could be indexed in an array on the patient by population number? Thoughts? Not sure how this might work in situations like on Dogpark, where patients are calculated against every measure (I don't quite recall how things work there).

I'm not sure about cached results actually.

J.B. was going to set up a meeting for Bonnie folks where I can go over this PR in detail. Perhaps we can add this discussion, as there are a few ways we can do this (and I'd like to know what people think makes sense for them).

Reply:
Want to go over it after burn down? I didn't want to add another meeting to people's calendars.

Reply:
That works, but it should be a meeting for the folks that need to approve it. Everyone else can be optional.